packages/web-runtime/src/store/user.js

```diff
@@ -172,6 +160,15 @@ const actions = {
       vueAuthInstance.events().addUserUnloaded(() => {
         console.log('user unloaded…')
         context.dispatch('cleanUpLoginState')
+
+        if (context.getters?.configuration?.auth) {
+          // => OAuth2
+          if (context.getters?.configuration?.auth?.logoutUrl) {
+            return (window.location = context.getters?.configuration?.auth?.logoutUrl)
+          } else if (context.getters?.configuration?.server) {
+            return (window.location = `${context.getters?.configuration?.server}index.php/logout`)
```

Comment:
Also made sure that there is no double `/` in the logout url here. the `server` url already is ensured to have a trailing slash.